addons/l10n_th/views/res_partner_views.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">res.partner.form.inherit.l10n_th</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <field name="vat" position="after">
+                <field name="l10n_th_company_info" string="Company" widget="radio" options="{'horizontal': True}" />
```

Comment:
This field might still be redundant

Reply:
Branch number is still an important information at least for Thaliand since it is a note for the clients (receivers) who is issuing the document (like report invoice)